<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="m_getHttpData">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>DownFiles</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="m_progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>471</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QDateEdit" name="m_startDate">
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>2</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::DaySection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyyMMdd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
    <widget class="QDateEdit" name="m_endDate">
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>2</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::DaySection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyyMMdd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="m_StudyModality">
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ES</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>US</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="m_getStudyImages">
     <property name="text">
      <string>StudyImage</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>199</y>
      <width>491</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="m_URL">
       <property name="text">
        <string>http://127.0.0.1:8080</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_studyuid">
       <property name="text">
        <string>1.2.826.1.1.3680043.2.461.20090916105245.168977.200909160196</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_seruid">
       <property name="text">
        <string>1.2.840.113619.2.55.3.604688119.969.1252951290.810.4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_imageuid">
       <property name="text">
        <string>1.2.840.113619.2.55.3.604688119.969.1252951290.968.37</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
